packages/main/test/specs/ComboBox.spec.js

```diff
@@ -685,6 +686,9 @@ describe("Accessibility", async () => {
 		assert.strictEqual(await invisibleMessageSpan.getHTML(false), itemAnnouncement1, "Span value is correct.")
 
 		await input.keys("ArrowDown");
+
+		assert.strictEqual(await invisibleMessageSpan.getHTML(false), itemAnnouncement3, "Span value is correct.")
```

Comment:
I would add a more meaningful description it would be easier to debug if a test fails.

Reply:
Done